rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+
+As a data-wrangler, I want to be able to succinctly enter the metadata I receive from the data contributor and have a clear understanding of what I need to do to ensure correct processing.
+
+As a schema developer, I want to ensure we are capturing and requiring, the metadata from a contributor to ensuring the pipelines can run appropriately.
+
+As a member of the pipelines team, I want to ensure that the pipelines process the correct datasets and I can easily access the metadata required to trigger the pipelines.  I do not want to be restricted in designing analysis based on how data was submitted.
+
+As a data consumer, I want to be able to find all data files that need to be processed together so that I can run my data processing pipeline.   I do not want to be restricted in designing analysis based on how data was submitted.
+
+As a data consumer, I want to be confident that the HCA DCP has correctly processed all data files that belong together so that I know the matrices I receive have been generated correctly.
+
+## Detailed Design
+
+### Component Notification Changes
+This RFC proposes no longer using bundle-level search-based notifications on primary bundles for initiating pipelines. Instead, we propose that a **specific trigger** is used to initiate analysis pipelines. This document lists the requirements for such notifications to be actionable by analysis and makes recommendations on overall implementation paradigm with specific examples for 10X and SS2 datasets.
+
+### Notification Requirements
+For a notification to be actionable by analysis, it must fulfill specific requirements. Specifically, the notification must include information about the bundles/data that belong to the same sequencing set and potentially need to be co-processed.
+
+We propose the term **Data Aggregation and Processing Set (DAPS)** to encompass all the different alternative representations of such sets. The only requirement imposed on such these representations is to be reproducible and resolvable to an immutable sets of data. Example representations of DAPS can be:
+
+*   A listing of all relevant bundles along with versions
+*   A search query accompanied by a timestamp (to guarantee reproducibility and immutability)
+*   Other future representations as project needs evolve (e.g. linking to a particular project bundle version)
+
+Critically, the submitted DAPS will not explicitly dictate the files that have to be co-processed in pipeline instances as this would require specialized knowledge of the pipelines by the submitter. Instead, **a DAPS indicates that aggregation of data must be performed only from the list of data provided (submission envelope), but the actual aggregation is delegated to the process of analysis initiation, subject to well defined modality-specific metadata requirements**. 
```

Comment:
How are the rules about how DAPS should be created defined? 

When are those rules invoked to create DAPS?

How does the introduction of DAPS change the data lifecycle in the DCP, a comparative diagram would be really helpful here.

Reply:
This all needs to be fleshed out in the next pass, both in the conceptual model and concrete examples,

Reply:
I will look forward to seeing the improved conceptual and concrete examples